restructuredtext_lint/test/test.py

```diff
@@ -93,7 +97,36 @@ def test_invalid_link(self):
 
         This is a regression test for https://github.com/twolfson/restructuredtext-lint/issues/12
         """
-        filepath = __dir__ + '/test_files/invalid_link.rst'
+        filepath = os.path.join(__dir__, 'test_files', 'invalid_link.rst')
         errors = restructuredtext_lint.lint_file(filepath)
         self.assertIn('Anonymous hyperlink mismatch: 1 references but 0 targets.', errors[0].message)
         self.assertIn('Hyperlink target "hello" is not referenced.', errors[1].message)
+
+
+class TestRestructuredtextLintCLI(TestCase):
+    """ Tests for 'rst-lint' CLI comand """
+
+    def test_rst_lint_filepaths_not_given(self):
+        """The `rst-lint` command is available and prints error if no filepath was given."""
+        with self.assertRaises(CalledProcessError):
+            output = check_output((sys.executable, rst_lint_path))
+            self.assertIn('too few arguments', output)
+
+    def test_rst_lint_correct_file(self):
+        """The `rst-lint` command prints out 'X is clean' if rst file is correct."""
+        raw_output = check_output((sys.executable, rst_lint_path, valid_rst))
+        output = str(raw_output).splitlines()
+        self.assertIn('{filepath} is clean'.format(filepath=valid_rst), output[0])
+        self.assertEqual(len(output), 1)
+
+    def test_rst_lint_many_files(self):
+        """The `rst-lint` command accepts many rst file paths and prints respective information for each of them."""
+        with self.assertRaises(CalledProcessError) as e:
+            check_output((sys.executable, rst_lint_path, valid_rst, invalid_rst))
```

Comment:
It would be nice to provide an example of what this executes but I can add that on while landing